@xen-orchestra/backups/_runners/VmsXapi.mjs

```diff
@@ -124,19 +126,21 @@ export const VmsXapi = class VmsXapiBackupRunner extends Abstract {
           nTriesByVmId[vmUuid]++
 
           const vmSettings = { ...settings, ...allSettings[vmUuid] }
-          const taskStart = { name: 'backup VM', data: { type: 'VM', id: vmUuid } }
-          const isLastRun = nTriesByVmId[vmUuid] === vmSettings.nRetriesVmBackupFailures + 1
+          const taskStart = { properties: { id: vmUuid, name: 'backup VM', type: 'VM' } }
+          const isLastRun = nTriesByVmId[vmUuid] === vmSettings.nRetriesVmBackupFailures + 2
 
           return this._getRecord('VM', vmUuid).then(
             disposableVm =>
               Disposable.use(disposableVm, async vm => {
-                if (taskStart.data.name_label === undefined) {
-                  taskStart.data.name_label = vm.name_label
+                if (taskStart.properties.name_label === undefined) {
+                  taskStart.properties.name_label = vm.name_label
                 }
 
                 const task = getVmTask()
+                // error has to be caught in the taskto prevent in failure, but handled outside the task to execute another task.run()
```

Comment:
```suggestion
                // error has to be caught in the task to prevent its failure, but handled outside the task to execute another task.run()
```